cg/services/order_validation_service/workflows/tomte/validation_rules.py

```diff
@@ -27,19 +29,19 @@
     validate_wells_contain_at_most_one_sample,
 )
 
-TOMTE_ORDER_RULES = [
+TOMTE_ORDER_RULES: list[Callable] = [
```

Comment:
```suggestion
TOMTE_ORDER_RULES: list[callable] = [
```
I think the one from typing is no longer required, the lower case one should work

Reply:
Cool, I'll swap it out then! 🥳